keylime/cmd/convert_config.py

```diff
@@ -357,6 +357,7 @@ def strip_quotes(config: RawConfigParser) -> None:
 def process_full_mapping(
     mapping: Dict[str, Any],
     old_config: RawConfigParser,
+    use_defaults: bool,
```

Comment:
seems that the tests needs to be updated to use this new parameter?

Reply:
Yes. I'll make this a WIP and fix the issues in tests

Reply:
This should be solved now